instance/static/html/instance/index.html

```diff
@@ -50,7 +50,13 @@
                 |
                 <span>Studio: <a href="{{ selected.instance.studio_url }}" target="_new">{{ selected.instance.studio_url }}</a></span>
               </li>
-              <li><a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> | {{ selected.instance.branch_name }}</li>
+              <li>
+                <a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> {{ selected.instance.branch_name }}
+                {% if selected.instance.github_pr_url %}
+                | <a href="{{ selected.instance.github_pr_url }}" target="_new">PR#{{ selected.instance.github_pr_number }}</a>
+                {% endif %}
+              </li>
+              {% endif %}
```

Comment:
@smarnach : )

Out of curiosity, what were the issues you had to setup the dev environment? If there are some gotchas other might run into, it could be worth adding a note or fixing them.